docs/pages/access-controls/access-lists/guide.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Getting Started with Access Lists
+description: Learn how to use Access Lists to manage and audit long lived access to Teleport managed resources.
+---
+
+This guide will help you:
+- Create an access list
+- Assign a member to it
+- Verify permissions granted through the list membership
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A running Teleport cluster.
+- A user with the preset `editor` role, which will have permissions to create Access Lists.
+
+### Step 1/4. Setting up an application service on the cluster for testing
+
+One of the easiest ways to get resources on the cluster for testing is to set up an application service
+with the debugging application enabled. To do this, add the following config to your `teleport.yaml`
+configuration:
+
+```yaml
+app_service:
+  enabled: yes
+  debug_app: true
+```
+
+And restart Teleport. The "dumper" app should show up in the resource list.
+
+![Debug app](../../../img/access-controls/access-lists/debug-app.png)
+
+## Step 2/4. Create a test user
+
+We need to create a simple test user that has only the `requester` role, which has no default access
+to anything within a cluster. This user will only be used for the purposes of this guide, so you may use
+another user if you so choose. If you would rather use your own user, skip to the next step.
+
+Navigate to the management pane and select "Users." Click on "Create New User" and fill in `test-user` as
+the name and select `requester` as the role.
+
+![Create Test User](../../../img/access-controls/access-lists/create-test-user.png)
+
+Click "Save," and then navigate to the provided URL in order to set up the credentials for your test user.
+Try logging into the cluster with the test user to verify that no resources show up in the resources page.
+
+## Step 3/4. Create an Access List
+
+Next, we'll create a simple access list that will grant the `access` role to its members.
+Login as the administrative user mentioned in the prerequisites. Navigate to the management pane and
+click on access lists. Click on "Create an Access List."
```

Comment:
```suggestion
click on access lists. Click on "Create an Access List".
```

Reply:
It's actually considered more proper to put the period inside the quotes. https://www.hamilton.edu/academics/centers/writing/style/essentials/punctuation-of-quotations#:~:text=The%20final%20period%20or%20comma,the%20period%20follows%20the%20citation.

Reply:
Huh, well, I can't argue with an .edu link 😄